Release notes for 8.14.1 (#16212)
elastic/logstash: docs/static/releasenotes.asciidoc

* <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>

[[logstash-8-14-1]]
=== Logstash 8.14.1 Release Notes

* Handle non-unicode payload in Logstash. (#16072) https://github.com/elastic/logstash/pull/16168[#16168]
* Backport PR #16176 to 8.14: PQ: avoid blocking writer when precisely full https://github.com/elastic/logstash/pull/16178[#16178]
* Revert pr #16050 https://github.com/elastic/logstash/pull/16201[#16201]
* Backport PR #16201 to 8.14: Revert pr #16050 https://github.com/elastic/logstash/pull/16203[#16203]
* Pin rexml gem version to 3.2.6 https://github.com/elastic/logstash/pull/16209[#16209]
* Backport PR #16209 to 8.14: Pin rexml gem version to 3.2.6 https://github.com/elastic/logstash/pull/16210[#16210]

==== Plugins

> *Elastic_integration Filter - 0.1.9*
> 
> 404: Not Found

Reviewer: ```suggestion
*Elastic_integration Filter - 0.1.9*

  - [DOC] Removes Tech Preview label and adds link to extending integrations topic in LSR https://github.com/elastic/logstash-filter-elastic_integration/pull/142[#142]

```

* <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>

[[logstash-8-14-1]]
=== Logstash 8.14.1 Release Notes

> 
> ---------- GENERATED CONTENT STARTS HERE ------------
> === Logstash Pull Requests with label v8.14.1

Reviewer: ```suggestion
```

✂️

> * Handle non-unicode payload in Logstash. (#16072) https://github.com/elastic/logstash/pull/16168[#16168]

Reviewer: ```suggestion
* Fixes JSON serialization of payloads that are not UTF-8, eliminating an issue where the Elasticsearch Output could get stuck in a retry loop (#16072) https://github.com/elastic/logstash/pull/16168[#16168]
```

> * Backport PR #16176 to 8.14: PQ: avoid blocking writer when precisely full https://github.com/elastic/logstash/pull/16178[#16178]

Reviewer: ```suggestion
* Fixes Persistent Queue bug in which a PQ configured with `queue.max_bytes` equal to its `queue.page_capacity` could become permanently blocked when _precisely_ full https://github.com/elastic/logstash/pull/16178[#16178]
```

> * Revert pr #16050 https://github.com/elastic/logstash/pull/16201[#16201]
> * Backport PR #16201 to 8.14: Revert pr #16050 https://github.com/elastic/logstash/pull/16203[#16203]

Reviewer: ```suggestion
* Fixes a regression in multi-local pipeline loader that caused variable-references in a configured `pipelines.yml` to not be replaced by their values in the environment and/or keystore https://github.com/elastic/logstash/pull/16201[#16201]
```

> * Pin rexml gem version to 3.2.6 https://github.com/elastic/logstash/pull/16209[#16209]
> * Backport PR #16209 to 8.14: Pin rexml gem version to 3.2.6 https://github.com/elastic/logstash/pull/16210[#16210]

Reviewer: ```suggestion
```

Tagged `[rn:skip]`

> === Logstash Commits between 8.14 and 8.14.0
> 
> Computed with "git log --pretty=format:'%h -%d %s (%cr) <%an>' --abbrev-commit --date=relative v8.14.0..8.14"
> 
> 8fa13707f - (HEAD -> 8.14, origin/8.14) Update patch plugin versions in gemfile lock (#16211) (2 hours ago) <github-actions[bot]>
> 224421f3e - Pin rexml gem version to 3.2.6 (#16209) (#16210) (3 hours ago) <github-actions[bot]>
> 550e93583 - Revert PR #16050 (#16203) (4 days ago) <github-actions[bot]>
> 7afb42c13 - Bump version to 8.14.1 (#16196) (5 days ago) <Rob Bavey>
> 78fb37928 - Release notes for 8.14.0 (#16155) (6 days ago) <github-actions[bot]>
> 
> === Logstash Plugin Release Changelogs ===
> Computed from "git diff v8.14.0..8.14 *.release"
> Changed plugin versions:
> logstash-filter-elastic_integration: 0.1.8 -> 0.1.9
> logstash-input-azure_event_hubs: 1.4.5 -> 1.4.7
> logstash-integration-jdbc: 5.4.10 -> 5.4.11
> logstash-integration-kafka: 11.4.1 -> 11.4.2
> logstash-output-http: 5.6.0 -> 5.6.1
> logstash-output-tcp: 6.2.0 -> 6.2.1
> ---------- GENERATED CONTENT ENDS HERE ------------

Reviewer: ```suggestion
```
✂️
